util/test/sub_clean

```diff
@@ -53,19 +53,19 @@ def cleanChapelTest(f):
             pass
         cleanCleanfiles(None, execname+'.cleanfiles', False)
 
-# Clean files listed in the file f
-def cleanCleanfiles(f, c, rmCore):
-    if f:
-        cleanfile = f+'/'+c
+# Clean files listed in the file c, in the directory d (if given)
+def cleanCleanfiles(d, c, rmCore):
+    if d:
+        cleanfile = d+'/'+c
     else:
         cleanfile = c
     if os.access(cleanfile,os.R_OK):
         cleanfiles = ReadCleanfiles(cleanfile)
         if rmCore:
             cleanfiles += ['core']
         for file in cleanfiles:
-            if f:
-                globfiles = glob.glob(f+'/'+file)
+            if d:
+                globfiles = glob.glob(d+'/'+file)
```

Comment:
Any interest in switching things like `d+'/'+file` over to `os.path.join(d, file)` while you're here?